frontend/src/components/avatar/index.tsx

```diff
@@ -1,9 +1,22 @@
 import { component$ } from '@builder.io/qwik'
+import type { Account } from '~/types'
 
 type Props = {
-	src: string
+	primary: Account
+	secondary: Account | null
 }
 
-export const Avatar = component$<Props>(({ src }) => {
-	return <img class="rounded h-12 w-12" src={src} />
+export const Avatar = component$<Props>(({ primary, secondary: second }) => {
+	return (
+		<div class={`relative ${second && 'pr-2 pb-2'}`}>
+			<a href={primary.url}>
+				<img class="rounded h-12 w-12" src={primary.avatar} />
```

Comment:
```suggestion
				<img class="rounded h-12 w-12" src={primary.avatar} alt={`Avatar of ${primary.display_name}`}/>
```

Reply:
I made the mistake of writing this code off a really old `main` 😢 
Rebasing fixed this 😄